Octokit.Tests/Clients/GistsClientTests.cs

```diff
@@ -1,321 +1,573 @@
 ﻿using NSubstitute;
-using Octokit;
 using Octokit.Internal;
 using System;
 using System.Collections.Generic;
 using System.Net;
 using System.Threading.Tasks;
 using Xunit;
 
-public class GistsClientTests
+namespace Octokit.Tests.Clients
```

Comment:
We're having an ongoing discussion about dropping namespaces from all our test classes - https://github.com/octokit/octokit.net/issues/1208 - it kind of goes against the standards for .NET but we think it has some benefits in places like the test runner output.

Are you able to drop this namespace and re-indent this file? It should simplify this diff a bit as well...